template-files/cheatsheet_example.tex

```diff
@@ -101,12 +105,12 @@ \section{Physik}
 \end{symbolbox}
 
 \begin{sectionbox}
-	\subsection{Einheitpräfixe}
-	\begin{tablebox}{l | ccccccccc}
-		$10^\pm$ 	& $21$ & $18$ & $15$ 	&  $12$ & $9$ &  $6$ & $3$ &  $2$ & $1$ \\ \cmrule
-		$+$			& $\underset{\ir zetta}{\si{\zetta}}$ & $\underset{\ir exa}{\si{\exa}}$ & $\underset{\ir peta}{\si{\peta}}$	& $\underset{\ir tera}{\si{\tera}}$ & $\underset{\ir giga}{\si{\giga}}$ & $\underset{\ir mega}{\si{\mega}}$ & $\underset{\ir kilo}{\si{\kilo}}$ & $\underset{\ir hecto}{\si{\hecto}}$ & $\underset{\ir deca}{\si{\deca}}$ \\
-		$-$ 		& $\underset{\ir zepto}{\si{\zepto}}$ & $\underset{\ir atto}{\si{\atto}}$ & $\underset{\ir femto}{\si{\femto}}$ & $\underset{\ir pico}{\si{\pico}}$ & $\underset{\ir nano}{\si{\nano}}$ & $\underset{\ir micro}{\si{\micro}}$ & $\underset{\ir milli}{\si{\milli}}$  & $\underset{\ir centi}{\si{\centi}}$& $\underset{\ir deci}{\si{\deci}}$
-	\end{tablebox}
+    \subsection{Einheitpräfixe}
+    \begin{tablebox}{l | ccccccccc}
+        $10^\pm$    & $21$ & $18$ & $15$    &  $12$ & $9$ &  $6$ & $3$ &  $2$ & $1$ \\ \cmrule
+        $+$            & $\underset{\ir zetta}{\si{\text{\zetta}}}$ & $\underset{\ir exa}{\si{\text{\exa}}}$ & $\underset{\ir peta}{\si{\text{\peta}}}$    & $\underset{\ir tera}{\si{\text{\tera}}}$ & $\underset{\ir giga}{\si{\text{\giga}}}$ & $\underset{\ir mega}{\si{\text{\mega}}}$ & $\underset{\ir kilo}{\text{kilo}}$ & $\underset{\ir hecto}{\si{\text{\hecto}}}$ & $\underset{\ir deca}{\si{\text{\deca}}}$ \\
```

Comment:
Why do we need all the `\text`? Shouldn't they be in math mode?

Reply:
I tried to build the pdf and got an error saying `Found prefix part with no unit.`. I think that comes from an update of the package that includes `\si` and it now requires you to write a unit after the prefix aswell and will throw an error if a prefix doesn't have a unit following. This is not a nice fix I know but I didn't find anything better.